wgsl/index.bs

```diff
@@ -529,7 +530,7 @@ The following types are <dfn dfn noexport>host-shareable</dfn>:
 * [=numeric vector=] types
 * [[#matrix-types]]
 * [[#array-types]] if the array type has a [=stride=] attribute and its element type is host-shareable
-* [[#struct-types]] if each member is host-shareable and has an [=offset=] attribute
+* [[#struct-types]] if each member is host-shareable and, if the size of the member isn't the same as the size of the type, has a [=span=] attribute
```

Comment:
This is a little bit confusing: "the size of the member isn't the same as the size of the type". Are we really talking about the *size* of the member here? Or maybe about it's memory span?

Reply:
Actually, we should have a separate definition somewhere, defining a set of types that match this description

Reply:
Memory span. Number of bytes taken up by the thing in memory. Will attempt to clarify.

Reply:
Better?